mindsdb_native/libs/backends/lightwood.py

```diff
@@ -200,27 +197,25 @@ def train(self):
 
         lightwood_config = self._create_lightwood_config()
 
-        if self.transaction.lmd['skip_model_training'] == True:
-            self.predictor = lightwood.Predictor(load_from_path=os.path.join(CONFIG.MINDSDB_STORAGE_PATH, self.transaction.lmd['name'] + '_lightwood_data'))
-        else:
-            self.predictor = lightwood.Predictor(lightwood_config)
 
-            # Evaluate less often for larger datasets and vice-versa
-            eval_every_x_epochs = int(round(1 * pow(10,6) * (1/len(train_df))))
+        self.predictor = lightwood.Predictor(lightwood_config)
 
-            # Within some limits
-            if eval_every_x_epochs > 200:
-                eval_every_x_epochs = 200
-            if eval_every_x_epochs < 3:
-                eval_every_x_epochs = 3
+        # Evaluate less often for larger datasets and vice-versa
+        eval_every_x_epochs = int(round(1 * pow(10,6) * (1/len(train_df))))
 
-            logging.getLogger().setLevel(logging.DEBUG)
-            if self.transaction.lmd['stop_training_in_x_seconds'] is None:
-                self.predictor.learn(from_data=train_df, test_data=test_df, callback_on_iter=self.callback_on_iter, eval_every_x_epochs=eval_every_x_epochs)
-            else:
-                self.predictor.learn(from_data=train_df, test_data=test_df, stop_training_after_seconds=self.transaction.lmd['stop_training_in_x_seconds'], callback_on_iter=self.callback_on_iter, eval_every_x_epochs=eval_every_x_epochs)
+        # Within some limits
+        if eval_every_x_epochs > 200:
+            eval_every_x_epochs = 200
+        if eval_every_x_epochs < 3:
+            eval_every_x_epochs = 3
+
+        logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
Some leftover debugging stuff it seems

Reply:
Nope, there for a reason related to lightwood logging, can't remember what but I'm fairly sure it's relevant or at least not there by mistake at the time it was placed

Reply:
is the documentation for this updated?